src/constants/data.ts

```diff
@@ -63,7 +63,7 @@ export interface NextExercise {
 }
 
 export type NextExerciseData = NextExercise & {
-  documents: Document[]
+  vocabularyItems: VocabularyItem[]
```

Comment:
I think the constant DOCUMENT_TYPES later in this file should also be renamed (I can't put the comment there, sorry)

Reply:
I also thought about it, but I think same as document_image, it's related to the API isn't it

Reply:
I agree here, I think it should be renamed. It is just used for saving in the local async storage, not for the API. And just renaming a var does not matter as long as it is not about a url or parameter passed to the api.